ethcore/private-tx/src/lib.rs

```diff
@@ -25,27 +25,27 @@ mod private_transactions;
 mod messages;
 mod error;
 
+extern crate common_types as types;
```

Comment:
Would it make sense to always use `common_types` (or always `types`) for consistency?

Reply:
Sounds good to me! Maybe then `s/extern crate common_types;/extern crate common_types as types;/` ?